packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -1,18 +1,120 @@
 <template>
     <div
-        :class="$style['c-mediaElement']"
-        data-test-id="mediaElement-component">
-        {{ componentName }}
+        data-test-id="mediaElement-component"
+        :class="[$style['c-mediaElement'], { [$style['c-mediaElement--stack']]: stacked }, { [$style['c-mediaElement--reverse']]: reverse }]">
+        <div :class="[$style['c-mediaElement-content'], contentAlignClass]">
+            <h3 :class="[$style['c-mediaElement-title'], $style['c-mediaElement-text'], textSizeClass]">
+                {{ title }}
+            </h3>
+            <p :class="[$style['c-mediaElement-text'], textSizeClass]">
```

Comment:
Similar to above – I'd put these on separate lines (we tend to do this in other components)

Reply:
agreed

Reply:
fixed